rpgJavaInterpreter-core/src/test/resources/smeup/MUDRNRAPU00105.rpgle

```diff
@@ -0,0 +1,35 @@
+     V* ==============================================================
+     V* 07/08/2024 APU001 Creation
+     V* ==============================================================
+    O * PROGRAM GOAL
+    O * Runtime evaluation of `£40ANM`, which uses
+    O *  `%ELEM(£40A)` for its initialization, during the evaluation
+    O *  of `£40A` `PLIST` parameter.
+     V* ==============================================================
+    O * JARIKO ANOMALY
+    O * Before the fix, the error occurred was `Void value for £40A`.
+     V* ==============================================================
+     D £40A            S             15    DIM(300)
+     D £40B            S             12    DIM(300)
+
+     D £40ANM          S              5S 0 INZ(%ELEM(£40A))                       Jariko Runtime Error: 'Void value for £40A' during evaluation of PLIST
```

Comment:
A comment here where you say: "runtime error during invocation of %ELEM(£40A)" is sufficient, as I think the rest does not add much to the test understanding.